sql/src/test/resources/calcite/tests/window/wikipediaScanWindow.sqlTest

```diff
@@ -0,0 +1,23 @@
+type: "operatorValidation"
+
+sql: |
+  SELECT
+    __time,
+    "user",
+    page,
+    LAG(page, 1) OVER (PARTITION BY "user" ORDER BY __time) as priorPage
+  FROM wikipedia
+
+expectedOperators:
+  - { type: "naiveSort", columns: [ { column: "user", direction: "ASC" }, { column: "__time", direction: "ASC" } ]}
+  - { type: "naivePartition",  partitionColumns: [ "user" ] }
+  - type: "window"
+    processor:
+      type: "offset"
+      inputColumn: page
+      outputColumn: w0
+      offset: -1
+
+# Not correct: there should actually be results here. Therefore, currently, this test only verifies that the
+# query is planned as expected, not that the results are correct.
+expectedResults: []
```

Comment:
Is the test not failing?  I would expect this test to be failing saying that the results are different.  If we want plan-only tests, I think I'd want the `type` to become `operatorPlanValidation` and then the test harness to only look at the expected operators and ignore the results.

Reply:
It is not failing. I am not sure why the results are empty. The native query looks good to me so I figured something was wrong with the execution part. I looked into it a little but in that time wasn't able to figure out where the results were getting dropped. One difference here is that the base query type is `scan`, all other tests have a base query type `groupBy`. Is that something that is supposed to work?

Reply:
Oooooh, yeah, scan doesn't work because it doesn't actually produce a good RowSignature for the ArrayListSegment (this is addressed some in https://github.com/apache/druid/pull/13773).  So that's likely the culprit.  Though, I would've expected that to generate an exception...